src/index.test.js

```diff
@@ -0,0 +1,9 @@
+import request from 'supertest';
+import app from './app';
+
+describe('Root', () => {
+  test('Returns a 200', async () => {
```

Comment:
This test name is accurate, and 100% understandable to a web developer, but we'd like test reports to be readily accessible to non-technical users. Something like `successfully returns the page` would work better.